front/app/modules/commercial/project_description_builder/admin/components/ContentViewer/Viewer/Viewer.test.tsx

```diff
@@ -49,21 +51,4 @@ describe('Preview', () => {
       screen.queryByTestId('projectDescriptionBuilderPreviewContent')
     ).not.toBeInTheDocument();
   });
-
-  it('should shows description when project description builder hook returns error', () => {
```

Comment:
We don't expect an error anymore since we only make the call when we know a project uses the content builder